services/powchain/src/main/java/tech/pegasys/teku/services/powchain/PowchainService.java

```diff
@@ -41,6 +43,7 @@
 
   private final Eth1DepositManager eth1DepositManager;
   private final Eth1HeadTracker headTracker;
+  private final Eth1StatusLogger eth1StatusLogger = new Eth1StatusLogger();
```

Comment:
Probably should make this just a local variable.  Ultimately we should only need to provide it to `ErrorTrackingEth1Provider`.